tests/plotting/test_colors.py

```diff
@@ -36,6 +36,11 @@
     COLORMAPS.append('batlow')
 
 
+@pytest.fixture(autouse=True)
+def skip_check_gc(skip_check_gc):
+    """GC check not needed."""
```

Comment:
Nit suggestion to shorten.
```suggestion
pytestmark = pytest.mark.usefixtures('skip_check_gc')
```